generator/internal/dart/templates/lib/message.mustache

```diff
@@ -28,6 +28,21 @@ class {{Codec.Name}} extends CloudMessage {
     this.{{Codec.Name}},
   {{/Fields}}
   }){{Codec.ConstructorBody}}
+
+  @override
+  {{#Codec.HasToStringLines}}
+  String toString() {
+    final contents = [
+      {{#Codec.ToStringLines}}
+      {{{.}}}
+      {{/Codec.ToStringLines}}
```

Comment:
I may have written this as a new property of the field (though maybe we do not have annotations for fields yet?), something like:

```suggestion
      {{#Fields}}
      {{{Codec.ToStringLine}}}
      {{/Fields}}
```

Reply:
As currently written, not all fields on a message contribute to the toString() for that message.

I'm thinking of the toString() as mostly being a 1-line summary of a message, and not something like a multi-line description of it and its contained messages. So, you'd have to manually iterate over child objects, ala:

```dart
  final result = await service.analyzeSentiment(
    AnalyzeSentimentRequest(document: document),
  );

  print('result: ${result}');
  print('documentSentiment: ${result.documentSentiment}');

  for (final sentence in result.sentences!) {
    print('');
    print('sentence:');
    print('  text: ${sentence.text}');
    print('  sentiment: ${sentence.sentiment}');
  }
```

Reply:
Ack.  Though presumably we will have a way to get all the details in a message (the JSON encoding?) so we can help folks troubleshoot problems?

Reply:
> Though presumably we will have a way to get all the details in a message (the JSON encoding?)

Yes, that makes sense. Perhaps they could rely on the toJson() method? Or, we could have a richer 'describe' style method, which would would be recursive. It could either return the entire description as a string, or, describe itself into some indented text structure.